semgrep-ci/teamcity/gitlab/settings.kts

```diff
@@ -0,0 +1,86 @@
+package _Self.buildTypes
+
+import jetbrains.buildServer.configs.kotlin.*
+import jetbrains.buildServer.configs.kotlin.buildFeatures.pullRequests
+import jetbrains.buildServer.configs.kotlin.buildSteps.ScriptBuildStep
+import jetbrains.buildServer.configs.kotlin.buildSteps.script
+import jetbrains.buildServer.configs.kotlin.triggers.schedule
+import jetbrains.buildServer.configs.kotlin.triggers.vcs
+
+object Security : BuildType({
+    name = "Security"
+
+    params {
+        password("env.SEMGREP_APP_TOKEN", "", display = ParameterDisplay.HIDDEN)
+        param("env.SEMGREP_REPO_URL", "%vcsroot.url%")
+        param("env.SEMGREP_REPO_NAME", "stuartcmehrens1/js-app-gitlab")
+        password("env.GITLAB_TOKEN", "", display = ParameterDisplay.HIDDEN)
+    }
+
+    vcs {
+        root(HttpsGitlabComStuartcmehrens1jsAppGitlabGitRefsHeadsMain)
```

Comment:
Is that the url to the gitlab repository?

Reply:
this was done when creating a connection to GitLab from the TeamCity UI: https://www.jetbrains.com/help/teamcity/configuring-vcs-roots.html.